dcspy/sdk/__init__.py

```diff
@@ -48,11 +48,18 @@ def load_dll(lib_type: DllSdk) -> Optional[Lib]:
             prog_files = environ['PROGRAMFILES']
         dll_path = f'{prog_files}\\Logitech Gaming Software\\SDK\\{lib_type.dir}\\{arch}\\Logitech{lib_type.name.capitalize()}.dll'
         LOG.debug(f'Selected DLL: {dll_path}')
-        ffi = FFI()
-        ffi.cdef(lib_type.header)
-        dll = ffi.dlopen(dll_path)
-        LOG.info(f'Loading of {lib_type.name} SDK success')
-        return dll
+        # use CDLL for Gkey SDK as it will be used with the ctypes library
+        # for the callback function
+        if lib_type.name == 'Gkey':
+            cdll = CDLL(dll_path)
```

Comment:
We can live with that for now. I mean one open issue we dot really need header `LogitechGkeyLib.h`, but it will allow us to keep common interface for `load_dll()` and all SDKs.